store/copr/coprocessor.go

```diff
@@ -993,7 +1036,19 @@ func (worker *copIteratorWorker) handleCopResponse(bo *Backoffer, rpcCtx *tikv.R
 		}
 	}
 	worker.sendToRespCh(resp, ch, true)
-	return nil, nil
+	if lastRange == nil {
+		return nil, nil
+	}
+	tasks, err := worker.buildCopTasksFromRemain(bo, lastRange, task)
```

Comment:
I'm afraid it's not efficient to call `buildCopTasks()` once for each paging request https://github.com/pingcap/tidb/issues/25726#issuecomment-875539350. You can slice `task.ranges` directly.